src/Tron/Protobuf/TronInternal.pb.h

```diff
@@ -13,7 +13,7 @@
 #error incompatible with your Protocol Buffer headers. Please update
 #error your headers.
 #endif
-#if 3007000 < PROTOBUF_MIN_PROTOC_VERSION
```

Comment:
could you please also revert Tron/Zilliqa pb.h?

Reply:
oh, I have merged the official master branch. Should I roll back?

Reply:
I have revert these file.